test/conftest.py

```diff
@@ -3,11 +3,17 @@
 import random
 import string
 
+from .constants import TEST_API_KEY
+
 
 @pytest.fixture(scope="function")
-def seam():
+def test_endpoint():
```

Comment:
We should name this fixture `server` and return `{ seed, endpoint }`: https://github.com/seamapi/javascript-http/blob/main/test/fixtures/seam/connect/api.ts (I should rename api.ts to server.ts 🤔 )

The seed can replace `constants.py` and can be defined in this file.

Seeing this, I'm realizing the fake servers should expose the seed on the API, i.e., at `/_fake/seed` (https://github.com/seamapi/fake-seam-connect/tree/main/src/pages/api/_fake), then the SDKs can just fetch the seed after starting the server.

Reply:
d9510f05ac8bbbac27898dbf744c976985bbbef3